acceptance/repository-cloudant/README.md

```diff
@@ -0,0 +1,77 @@
+# @loopback/test-repository-cloudant
+
+Acceptance tests for `@loopback/repository` + `loopback-connector-cloudant`.
+
+## Running the test suite
+
+### Using own Cloudant instance
+
+If you have a local Cloudant instance listening on `localhost` and the default
+port, with a `root` user and an empty password, use the following command:
+
+```bash
+npm test
+```
+
+If you have a local or remote Cloudant instance and would like to use that to
+run the test suite, use the following command:
+
+**Linux & MacOS**
+
+```bash
+CLOUDANT_URL=<URL> CLOUDANT_HOST=<HOST> CLOUDANT_PORT=<PORT> CLOUDANT_USER=<USER> CLOUDANT_PASSWORD=<PASSWORD> CLOUDANT_DATABASE=<DATABASE> npm test
+```
+
+**Windows**
+
+```bash
+SET CLOUDANT_HOST=<HOST>
+SET CLOUDANT_PORT=<PORT>
+SET CLOUDANT_USER=<USER>
+SET CLOUDANT_PASSWORD=<PASSWORD>
+SET CLOUDANT_DATABASE=<DATABASE>
+npm test
+```
+
+### Using Docker (Linux, MacOS, WSL)
+
+If you do not have a local Cloudant instance, you can also run the test suite
+with very minimal requirements. We use couchDB docker image to run the test
+locally.
+
+NOTICE: we use `couchDB3` docker image for testing Cloudant because Cloudant
+doesn't have a maintained image, and most of the their functionalities are the
+same (except couchDB doesn't support geosearch).
+
+- Install [Docker](https://docs.docker.com/engine/installation/).
+
+- Use the `couchDB3:latest` image, and rename the container as
+  `cloudant-testdb`.
+
+- Start the container, and create an admin and password for this image at
```

Comment:
Can we automate this and the previous step please? In other repository acceptance packages, the docker-based setup requires the developer to run a single script only. Can we perhaps tell people to use `npm docker:setup`? (See also my other comments suggesting to move that script to this package.)

Reply:
Done, and README is updated. ( the script is run in this folder now)